server/auth_server.go

```diff
@@ -487,14 +484,16 @@ func RefreshToken(json map[string]string) (model.Token, error) {
 		}
 
 		payload := make(map[string]interface{})
+		if provider.GetName() == "ldap" {
+			payload["external_id"] = token.ExternalAccountID
+		}
```

Comment:
@prachidamle Looks like it doesn't need this. Maybe I changed RefreshToken later and earlier the code was doing something where it needed a field in payload. Removing this